common/templates/general.go

```diff
@@ -227,11 +232,24 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 			if val == nil {
 				continue
 			}
-			embed, err := CreateEmbed(val)
-			if err != nil {
-				return nil, err
+			switch val.(type) {
+			case Slice, []*discordgo.MessageEmbed:
+				v, _ := indirect(reflect.ValueOf(val))
+				for i := 0; i < v.Len() && i < maxEmbeds; i++ {
+					embed, err := CreateEmbed(v.Index(i).Interface())
+					if err != nil {
+						return nil, err
+					}
+					embeds = append(embeds, embed)
+				}
+				msg.Embeds = embeds
+			default:
+				embed, err := CreateEmbed(val)
+				if err != nil {
+					return nil, err
+				}
+				msg.Embeds = []*discordgo.MessageEmbed{embed}
 			}
```

Comment:
Untested, but the following should provide more flexibility wrt the input type:
```suggestion
			v, _ := indirect(reflect.ValueOf(val))
			if v.Kind() == reflect.Slice {
				const maxEmbeds = 10 // Discord limitation
				for i := 0; i < v.Len() && i < maxEmbeds; i++ {
					embed, err := CreateEmbed(v.Index(i).Interface())
					if err != nil {
						return nil, err
					}
					msg.Embeds = append(msg.Embeds, embed)
				}
			} else {
				embed, err := CreateEmbed(val)
				if err != nil {
					return nil, err
				}
				msg.Embeds = []*discordgo.MessageEmbed{embed}
			}
```